src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1467,20 +1467,26 @@ private void DynamoViewModelRequestAboutWindow(DynamoViewModel model)
         private void DynamoViewModelRequestRequestPackageManagerPublish(PublishPackageViewModel model)
```

Comment:
Would you rename this function? e.g. `DynamoViewModelRequestPackageManager`

Reply:
Oopsie .. sure thing!